modules/nodes-adding-bare-metal-host-to-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-adding-bare-metal-host-to-cluster-using-web-console_{context}"]
+= Adding a bare metal host to the cluster using the web console
+
+You can add bare metal hosts to the cluster in the web console.
+
+.Prerequisites
+
+* Install a cluster on bare metal, and provision a new bare metal host for use in the cluster.
+* Install the {product-title} CLI (oc).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with Dialog*.
+. Specify a unique name for the new bare metal host.
+. Set the boot mac address.
+. Optionally, enable power management for the host. This allows {product-title} to control the power state of the host.
+. Enter the user credentials for the host baseboard management controller (BMC) device.
+. Select to Power on the host after creation, and select *Create*.
+
+[NOTE]
+====
+This procedure creates the bare metal host. In order to create a corresponding bare metal node, scale up the number of replicas to match the number of available bare metal hosts using the `oc scale` command and the appropriate bare metal machine set.
```

Comment:
since we are adding the BMH via web console, should we scale via web console too ?

Reply:
Agreed. Users can find the relevant MachineSet within the Web Console, switch to its YAML tab, and increase the number of Machine replicas it creates to n+1 there as well.

I sort of wish we would just do this automatically if the user adds a host via the UI, and I'm forgetting why we don't. Maybe we couldn't know for certain which MachineSet should be scaled? I think we should consult the experts in #forum-kni to find out how a user should determine which MachineSet to scale.

Reply:
See the `hostSelector` [here](https://github.com/openshift/cluster-api-provider-baremetal/blob/master/docs/api.md#baremetalmachineproviderspec). You need that to match the labels on the BMH. You'll notice it has a `matchExpressions` component which I wouldn't make much noise about. ;) Labels are the way to go.

Reply:
The auto-scaling you mention does exist and can be enabled with an annotation on the MachineSet. See [MachineSet Scaling](https://github.com/openshift/cluster-api-provider-baremetal/#machineset-scaling).

Reply:
@mhrivnak I think auto-scaling probably warrants documenting on it's own here. I'll add this.

Reply:
> See the `hostSelector` [here](https://github.com/openshift/cluster-api-provider-baremetal/blob/master/docs/api.md#baremetalmachineproviderspec). You need that to match the labels on the BMH. You'll notice it has a `matchExpressions` component which I wouldn't make much noise about. ;) Labels are the way to go.

https://github.com/openshift/openshift-docs/pull/30656/files#r617699273